irods/connection.py

```diff
@@ -4,19 +4,22 @@
 import struct
 import hashlib
 import six
-import struct
+# import struct
 import os
 import ssl
 
 
 from irods.message import (
-    iRODSMessage, StartupPack, AuthResponse, AuthChallenge,
+    iRODSMessage, StartupPack, AuthResponse, AuthChallenge, AuthPluginOut,
     OpenedDataObjRequest, FileSeekResponse, StringStringMap, VersionResponse,
-    GSIAuthMessage, ClientServerNegotiation, Error)
```

Comment:
The AuthMessage previously used for GSI is instead quite generic and also valid for PAM and other plugins

So that I renamed it as AuthPluginMessage